### PR TITLE
prediction update

### PR DESCRIPTION
**Pull request by brandondutra**
Updated the outputs of the graph.

local batch prediction: working
online batch prediction: import error
local prediction: working
online prediction: working

### CODE REVIEW

File: solutionbox/structured_data/datalab_solutions/structured_data/trainer/task.py [hunk 1]
```diff
@@ -145,62 +154,56 @@ def get_export_signature(examples, features, predictions):
...
+      #input_target_score = vector_slice(predictions, features[target_name])
+      
+      # get top k labels and their scores.
       (top_k_values, top_k_indices) = tf.nn.top_k(predictions, k=args.top_n)
       top_k_labels = tf.contrib.lookup.index_to_string(
           tf.to_int64(top_k_indices),
           mapping=string_value)
+   
+      # Write the top_k values using 2*top_k columns. 
+      num_digits = int(math.ceil(math.log(args.top_n, 10)))
```
Comment (qimingj) [inline solutionbox/structured_data/datalab_solutions/structured_data/trainer/task.py]: Can we force top_n < 100? The following logic seems complicated and hard to maintain. top_n >= 100 seems unreasonable already.

File: solutionbox/structured_data/datalab_solutions/structured_data/_package.py [hunk 2]
```diff
@@ -140,6 +145,8 @@ def cloud_preprocess(output_dir, input_feature_file, input_file_pattern=None, sc
     args.append('--bigquery_table=%s' % full_name)
   
   print('Starting cloud preprocessing.')
+  print('Track BigQuery status at')
+  print('https://bigquery.cloud.google.com/queries/%s' % _default_project())
```
Comment (brandondutra) [inline solutionbox/structured_data/datalab_solutions/structured_data/_package.py]: done

File: solutionbox/structured_data/datalab_solutions/structured_data/_package.py [hunk 3]
```diff
@@ -208,6 +222,9 @@ def cloud_train(train_file_pattern,
     transforms_file: File path to the transforms file.
     model_type: model type
     max_steps: Int. Number of training steps to perform.
+    top_n: Int. For classification problems, the output graph will contain the
```
Comment (brandondutra) [inline solutionbox/structured_data/datalab_solutions/structured_data/_package.py]: done

File: solutionbox/structured_data/datalab_solutions/structured_data/_package.py [hunk 4]
```diff
@@ -277,30 +296,89 @@ def cloud_train(train_file_pattern,
...
+                              header=None,
+                              names=header.split(','))
+
+    # Print any errors to the screen.
+    errors_file = glob.glob(os.path.join(tmp_dir, 'errors*'))
+    if errors_file and os.path.getsize(errors_file[0]) > 0:
+      print('Warning: there are errors. See below:')
+      with open(errors_file[0], 'r') as f:
+        text = f.read()
+        print(text)
```
Comment (brandondutra) [inline solutionbox/structured_data/datalab_solutions/structured_data/_package.py]: done

File: solutionbox/structured_data/datalab_solutions/structured_data/_package.py [hunk 5]
```diff
@@ -277,30 +296,89 @@ def cloud_train(train_file_pattern,
...
   """Use Online prediction.
 
   Runs online prediction in the cloud and prints the results to the screen. For
   running prediction on a large dataset or saving the results, run
   local_batch_prediction or batch_prediction.
 
   Args:
-
+    model_name: deployed model name
+    model_verion: depoyed model versions
```
Comment (brandondutra) [inline solutionbox/structured_data/datalab_solutions/structured_data/_package.py]: done

File: solutionbox/structured_data/datalab_solutions/structured_data/trainer/task.py (hunk 1 above)
Comment (brandondutra) [inline solutionbox/structured_data/datalab_solutions/structured_data/trainer/task.py]: No. 100 could be reasonable. Let the user decide.

**Pull request merged**